.siftlint.yml

```diff
@@ -0,0 +1,75 @@
+disabled_rules:
```

Comment:
You can remove this file 😄

Reply:
Thank you for the feedback. I will do so. 👍🏻